src/components/UserPortal/ChatRoom/ChatRoom.tsx

```
@@ -228,11 +236,100 @@
+        setAttachment(objectUrl);
+      } catch (error) {
+        console.error('Error uploading file:', error);
+        // TODO: Show error toast to user
+      }
     }
   };
 
+  // Clean up object URL when component unmounts or attachment changes
+  useEffect(() => {
+    return () => {
+      if (attachment && attachment.startsWith('blob:')) {
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)